### PR TITLE
Ticket list

### LINKED ISSUE #149: Create Ticket List page
Page 20 of the mockups https://drive.google.com/file/d/1YqboQogczYm1HkyRqEtVSzeQ61T9hWU2/view

### PR DESCRIPTION
**Pull request by zfregin**
Basic table layout to display tickets

### CODE REVIEW

File: src/views/tickets.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+            <UserContext.Consumer>
+                {session => {
+                    this.context = session;
+                    return (
+                        <div className="tickets__container">
+                            <div className="section-header">
+                                <h2 className="page-title">Tickets</h2>
+                            </div>
+                            <div className="search-section">
+                              <input class="input is-rounded" placeholder="Search by tenant, manager, property, or JOIN staff"></input>
```
Comment (Chris-Boe) [inline src/views/tickets.js:117]: don't forget to use "className" throughout this file instead of "class" so react can use it

**Review by Chris-Boe — CHANGES_REQUESTED** [review]
Looks good so far! Had a few issues with class vs className to fix. I didn't test with any real data, since Hugh is working on the backend endpoints for Tickets currently, but if anything breaks we can come back to it when that gets in!
I noticed the table doesn't fill the page horizontally like in the mockup, but the style fixes might end up fixing that: 
image

File: src/views/tickets.js (hunk 1 above)
Comment (zfregin) [reply src/views/tickets.js:117]: Yeah, I just copy pasted from Bulma and forgot to make the change. Sounds like we will be building out some of these primitives ourselves though

File: src/views/tickets.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+        this.getTickets = this.getTickets.bind(this)
+    }    
+
+    componentDidMount() {
+        this.getTickets(this.context);
+    }
+
+    getTickets = (context) => {
+        axios.get(`${process.env.REACT_APP_API_URL}/properties`, { headers: {"Authorization" : `Bearer ${context.user.accessJwt}`} })
```
Comment (KentShikama) [inline src/views/tickets.js]: Note the axios requests will need to look like `axios.get("/api/properties", ...)` (see https://github.com/codeforpdx/dwellingly-app/pull/177)

**Comment by Chris-Boe** [comment]
Let's also take out the changes to `dwellingly/package-lock.json` since that file no longer exists and we don't want to add it back in

**Review by Chris-Boe — APPROVED** [review]
👍

**Pull request merged**